tests/src/test/scala/system/packages/MessageHubFeedTests.scala

```diff
@@ -274,6 +274,67 @@ class MessageHubFeedTests
       assert(matchingActivations.length == 0)
   }
 
+  it should "return correct status and configuration" in withAssetCleaner(wskprops) {
+    val currentTime = s"${System.currentTimeMillis}"
+
+    (wp, assetHelper) =>
+      val triggerName = s"/_/dummyMessageHubTrigger-$currentTime"
+      println(s"Creating trigger ${triggerName}")
+
+      val username = kafkaUtils.getAsJson("user")
+      val password = kafkaUtils.getAsJson("password")
+      val admin_url = kafkaUtils.getAsJson("kafka_admin_url")
+      val brokers = kafkaUtils.getAsJson("brokers")
+
+      createTrigger(assetHelper, triggerName, parameters = Map(
+        "user" -> username,
+        "password" -> password,
+        "api_key" -> kafkaUtils.getAsJson("api_key"),
+        "kafka_admin_url" -> admin_url,
+        "kafka_brokers_sasl" -> brokers,
+        "topic" -> topic.toJson,
+        "isBinaryKey" -> false.toJson,
+        "isBinaryValue" -> false.toJson
+      ))
+
+      val run = wsk.action.invoke(actionName, parameters = Map(
+        "triggerName" -> triggerName.toJson,
+        "lifecycleEvent" -> "READ".toJson,
+        "authKey" -> wp.authKey.toJson
+      ))
+
+      withActivation(wsk.activation, run) {
+        activation =>
+          activation.response.success shouldBe true
+
+          inside (activation.response.result) {
+            case Some(result) =>
+              val config = result.getFields("config").head.asInstanceOf[JsObject].fields
+              val status = result.getFields("status").head.asInstanceOf[JsObject].fields
+
+              config should contain("brokers" -> brokers)
```

Comment:
You should be able build a JsObject that has all these fields then do one or two comparisons instead of several.

Reply:
Ex:
```
val expectedConfig = JsObject("isBinaryKey" -> false.toJson, "isBinaryValue" -> false.toJson, etc...)
config shouldBe(expectedConfig)
```

Reply:
that would work except for the fact the trigger name as returned is not equal to the trigger name passed in

Reply:
and i would still have to test fields individually for status as i cannot know the value of the dateChanged field

Reply:
What does the triggerName look like when it is returned here?

Reply:
passed in as `/_/dummyTrigger-...` returned as `/guest/dummyTrigger-...`

Reply:
So we know what the returned trigger name will be. That means the config property can be compared to a JsObject.

What does dateChanged look like? Is that a timestamp?

Reply:
I see two different possible values for dataChanged.

![screen shot 2017-10-17 at 10 33 53 am](https://user-images.githubusercontent.com/11520887/31670932-c4504d26-b326-11e7-82fe-e1b7de6dce20.png)

Reply:
no, there is no absolute guarantee of the namespace when this runs

Reply:
time in millis